prog/sensors/sensors.1

```diff
@@ -10,15 +10,15 @@
 .\" manual under the conditions for verbatim copying, provided that the
 .\" entire resulting derived work is distributed under the terms of a
 .\" permission notice identical to this one
-.\" 
```

Comment:
Any idea what changed here ? If possible please undo; this is an unrelated change.